From 5c11a11471e3aea547cedad63a550b16508a4c05 Mon Sep 17 00:00:00 2001
From: Vladimir Smirnov <civil@gentoo.org>
Date: Mon, 19 May 2014 19:50:12 +0400
Subject: [PATCH 05/30] Add headers for opensolaris

Thanks to the nice OpenSolaris guys, I finally figured out that on
OpenSolaris 11, the header dependencies have changed.

Patch-Name: gentoo/opensolars_headers.diff
Original-By: Torsten Veller <tove@gentoo.org>
---
 perl.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/perl.h b/perl.h
index 17a21a1c42..b0f6ab8d89 100644
--- a/perl.h
+++ b/perl.h
@@ -1769,6 +1769,8 @@ Localize C<$_>.  See L<perlguts/Localizing changes>.
  */
 
 #if defined(I_SYSMODE)
+#include <sys/stat.h>
+#include <sys/vnode.h>
 #include <sys/mode.h>
 #endif
 
-- 
2.26.3

